requirements-dev.txt

```diff
@@ -0,0 +1 @@
+poetry
```

Comment:
There may be a base container which includes `poetry` but `mcr.microsoft.com/devcontainers/python:0-3.10-bullseye` doesn't seem to.

This is also a place where you could fix versions of either poetry or other supporting tools. This could also be folded into the `postCreateCommand`.